Deergha.md

```diff
@@ -0,0 +1,27 @@
+# Deergha Worklog
+
+----
+##Till 29th Feb
+
+
+> Make Traceability Matrix Version 1
+
+> Survey analysis updated
+
+> Formatting of SRS
+
+> Took interview of MScIT student

```

Comment:
Deergha
you don't have to you \> for bullet points.
Please refer to markdown or other files on how to do bullet points.